orientdb/src/main/scala/akka/stream/alpakka/orientdb/scaladsl/OrientDBSink.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb.scaladsl
+
+import akka.Done
+import akka.stream.alpakka.orientdb._
+import akka.stream.scaladsl.{Keep, Sink}
+import com.orientechnologies.orient.core.record.impl.ODocument
+
+import scala.concurrent.Future
+
+object OrientDBSink {
+
+  /**
+   * Scala API: creates a sink based on [[OrientDBFlowStage]] that accepts as ODocument
```

Comment:
The `OrientDBFlowStage` won't show in the Scaladoc as it is private. (I know other connectors do the same thing.)